src/cloud-code/Parse.Cloud.js

```diff
@@ -440,132 +440,36 @@ ParseCloud.afterFind = function (parseClass, handler, validationHandler) {
   );
 };
 
-/**
- * Registers a before save file function.
- *
- * **Available in Cloud Code only.**
- *
- * ```
- * Parse.Cloud.beforeSaveFile(async (request) => {
- *   // code here
- * }, (request) => {
- *   // validation code here
- * });
- *
- * Parse.Cloud.beforeSaveFile(async (request) => {
- *   // code here
- * }, { ...validationObject });
- *```
- *
- * @method beforeSaveFile
- * @name Parse.Cloud.beforeSaveFile
- * @param {Function} func The function to run before saving a file. This function can be async and should take just one parameter, {@link Parse.Cloud.FileTriggerRequest}.
- * @param {(Object|Function)} validator An optional function to help validating cloud code. This function can be an async function and should take one parameter a {@link Parse.Cloud.FileTriggerRequest}, or a {@link Parse.Cloud.ValidatorObject}.
- */
 ParseCloud.beforeSaveFile = function (handler, validationHandler) {
-  validateValidator(validationHandler);
-  triggers.addFileTrigger(
-    triggers.Types.beforeSaveFile,
-    handler,
-    Parse.applicationId,
-    validationHandler
-  );
+  Deprecator.logRuntimeDeprecation({
+    usage: 'Parse.Cloud.beforeSaveFile',
+    solution: 'Use Parse.Cloud.beforeSave(Parse.File, (request) => {})',
+  });
+  ParseCloud.beforeSave(Parse.File, handler, validationHandler);
 };
 
-/**
- * Registers an after save file function.
- *
- * **Available in Cloud Code only.**
- *
- * ```
- * Parse.Cloud.afterSaveFile(async (request) => {
- *   // code here
- * }, (request) => {
- *   // validation code here
- * });
- *
- * Parse.Cloud.afterSaveFile(async (request) => {
- *  // code here
- * }, { ...validationObject });
- *```
- *
- * @method afterSaveFile
- * @name Parse.Cloud.afterSaveFile
- * @param {Function} func The function to run after saving a file. This function can be async and should take just one parameter, {@link Parse.Cloud.FileTriggerRequest}.
- * @param {(Object|Function)} validator An optional function to help validating cloud code. This function can be an async function and should take one parameter a {@link Parse.Cloud.FileTriggerRequest}, or a {@link Parse.Cloud.ValidatorObject}.
```

Comment:
This removes the API reference for deprecated triggers. I think they should be kept but flagged as deprecated. There is a JSDoc flag for that I believe. Otherwise the API will be undocumented.

Also, the description in existing triggers should be amended, because they only mention "Class" but it may not be obvious that Parse.File is also just another class name in that sense.